sap/note/sectSysctl.go

```diff
@@ -33,11 +33,9 @@ func OptSysctlVal(operator txtparser.Operator, key, actval, cfgval string) strin
 		fieldE := ""
 		if len(allFieldsC) != len(allFieldsE) {
 			fieldE = fieldC
-
-			if (operator == txtparser.OperatorLessThan || operator == txtparser.OperatorLessThanEqual) && k == 0 {
-				fieldE = allFieldsE[0]
-			}
-			if (operator == txtparser.OperatorMoreThan || operator == txtparser.OperatorMoreThanEqual) && k == len(allFieldsC)-1 {
+			isOperatorLTOrLTE := (operator == txtparser.OperatorLessThan ||
+				operator == txtparser.OperatorLessThanEqual)
+			if isOperatorLTOrLTE && (k == 0 || k == len(allFieldsC)-1) {
```

Comment:
This will give you now wrong results.
The first if statement is related to **Less** (<), but the second one is related to **More** (>) by intention.
To get the expected behavior you still need 2 if statements